src/test/java/com/google/api/generator/engine/writer/ImportWriterVisitorTest.java

```diff
@@ -363,6 +364,41 @@ public void writeVariableExprImports_nestedReference() {
         writerVisitor.write());
   }
 
+  @Test
+  public void writeVariableExprImports_withAnnotations() {
+    Variable variable =
+        Variable.builder()
+            .setName("expr")
+            .setType(TypeNode.withReference(ConcreteReference.withClazz(Expr.class)))
+            .build();
+
+    TypeNode fakeAnnotationType =
+        TypeNode.withReference(
+            VaporReference.builder().setName("FakeAnnotation").setPakkage("com.foo.bar").build());
+
+    AnnotationNode annotation =
+        AnnotationNode.builder()
+            .setType(fakeAnnotationType)
+            .setDescription(
+                ValueExpr.withValue(
+                    PrimitiveValue.builder().setValue("1").setType(TypeNode.INT).build()))
+            .build();
+
+    VariableExpr variableExpr =
+        VariableExpr.builder()
+            .setVariable(variable)
+            .setIsDecl(true)
+            .setAnnotations(Arrays.asList(annotation))
```

Comment:
Nit: extremely small point, might save just a few lines using the concrete `Generated` as test.
```suggestion=
            .setAnnotations(Arrays.asList(
                AnnotationNode.withType(
                    TypeNode.withReference(ConcreteReference.withClazz(Generated.class)))))
```

Reply:
Ohh I hadn't realized this - thank you for the suggestion and updated!